web/src/components/profile/ProfileAddressLockedFields.tsx

```diff
@@ -8,24 +10,40 @@ export const ProfileAddressLockedFields = (): ReactElement => {
   const Config = getMergedConfig();
   return (
     <div className="margin-top-4 margin-bottom-4" data-testid={"locked-profileAddressFields"}>
-      <div className="text-bold margin-bottom-05">
-        {Config.profileDefaults.fields.businessAddress.default.header}
+      <div className="flex flex-row fac margin-bottom-05">
+        <div className="text-bold">{Config.profileDefaults.fields.businessAddress.default.header}</div>
+        <div className="margin-left-1">
+          <ArrowTooltip
+            title={Config.profileDefaults.default.lockedFieldTooltipText}
+            data-testid="profile-business-address-locked-tooltip"
+          >
+            <div className="fdr fac font-body-lg">
+              <Icon>help_outline</Icon>
+            </div>
+          </ArrowTooltip>
+        </div>
       </div>
 
       {state.addressData.addressLine1 &&
-        state.addressData.addressMunicipality &&
-        state.addressData.addressState &&
-        state.addressData.addressZipCode && (
-          <>
-            <div data-testid={"locked-profileAddressLine1"}>{state.addressData.addressLine1}</div>
-            {state.addressData.addressLine2 && (
-              <div data-testid={"locked-profileAddressLine2"}>{state.addressData.addressLine2}</div>
-            )}
-            <div
-              data-testid={"locked-profileAddressMuniStateZip"}
-            >{`${state.addressData.addressMunicipality?.displayName}, ${state.addressData.addressState?.shortCode} ${state.addressData.addressZipCode}`}</div>
-          </>
-        )}
+      state.addressData.addressMunicipality &&
+      state.addressData.addressState &&
+      state.addressData.addressZipCode ? (
+        <>
+          <div data-testid={"locked-profileAddressLine1"}>{state.addressData.addressLine1}</div>
+          {state.addressData.addressLine2 && (
+            <div data-testid={"locked-profileAddressLine2"}>{state.addressData.addressLine2}</div>
+          )}
+          <div
+            data-testid={"locked-profileAddressMuniStateZip"}
+          >{`${state.addressData.addressMunicipality?.displayName}, ${state.addressData.addressState?.shortCode} ${state.addressData.addressZipCode}`}</div>
+        </>
+      ) : (
+        <>
+          <div data-testid="locked-profileAddressEmpty">
```

Comment:
same as previous: [dust] locked-profileAddressNotProvidedis another option and aligns better with the copy in the JSON and ~YAML` files. 
- especially truer here with the two (`locked-profileAddressEmpty` and `profileAddressNotProvided`) so close together

Reply:
im down for new names?